fftool/arg_parser.py

```
@@ -61,6 +49,12 @@ def arg_parser():
     )
 
     parser.add_argument(
```

Don't know if this is overkill, but when wifi goes down, we coul probably 'silently' check for that and proceed, but then the user might be thinking they're getting the latest browser (which isn't cool).  Might be better to let them fail, but then offer them the option to do --no-download (and run with cached).  Heck, it worked well on the plane :)